packages/ui-core/src/utils/mutateArray.ts

```diff
@@ -0,0 +1,16 @@
+const mutate = (array: any[], from: number, to: number) => {
```

Comment:
the same logic is used here `query-creator/src/utils/mutateArray.ts`

we should export these utils

Reply:
Yeap, I will import it in query creator.